MediaGallerySynchronizationApi/Api/SelectByBatchesGeneratorInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGallerySynchronizationApi\Api;
```

Comment:
Let's move this interface to the Model namespace as this is not an extension point, only a useful utility for internal implementation.
```suggestion
namespace Magento\MediaGallerySynchronizationApi\Model;
```